packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -256,49 +232,34 @@ export class RafikiService implements IRafikiService {
       wh.data,
       walletAddress
     )
-    const holdResult = await this.rapydClient.holdLiquidity({
-      amount: this.amountToNumber(amount),
-      currency: amount.assetCode,
-      ewallet: rapydWalletId
-    })
 
-    if (holdResult.status?.status !== 'SUCCESS') {
-      throw new Error(
-        `Unable to hold liquidity on wallet: ${rapydWalletId} on ${
-          EventType.OutgoingPaymentCreated
-        } error message: ${holdResult.status?.message || 'unknown'}`
-      )
-    }
     await this.rafikiClient.depositLiquidity(wh.id)
 
     this.logger.info(
       `Succesfully held ${this.amountToNumber(
         amount
-      )} in ${rapydWalletId}  on ${EventType.OutgoingPaymentCreated}`
+      )} in ${gateHubWalletId}  on ${EventType.OutgoingPaymentCreated}`
     )
   }
 
   private async handleOutgoingPaymentCompleted(wh: WebHook) {
     const walletAddress = await this.getWalletAddress(wh)
     const debitAmount = this.getAmountFromWebHook(wh)
 
-    const source_ewallet = await this.getRapydWalletId(walletAddress)
+    const { gateHubWalletId: sendingWallet, userId } =
+      await this.getGateHubWalletAddress(walletAddress)
 
     if (!this.validateAmount(debitAmount, wh.type)) {
       return
     }
 
-    await this.rapydClient.releaseLiquidity({
+    await this.gateHubClient.createTransaction({
       amount: this.amountToNumber(debitAmount),
-      currency: debitAmount.assetCode,
-      ewallet: source_ewallet
-    })
-
-    await this.rapydClient.transferLiquidity({
-      amount: this.amountToNumber(debitAmount),
-      currency: debitAmount.assetCode,
-      destination_ewallet: this.env.RAPYD_SETTLEMENT_EWALLET,
-      source_ewallet
+      vault_uuid: this.getVaultUuid(debitAmount.assetCode),
+      sending_address: sendingWallet,
+      receiving_address: this.env.GATEHUB_SETTLEMENT_WALLET_ADDRESS,
+      type: 'transfer',
```

Comment:
I think this should be the same as [here](https://github.com/interledger/testnet/pull/1612/files#r1765241966).